pkgs/games/multimc/default.nix

```diff
@@ -0,0 +1,97 @@
+{ lib, stdenv, bash
+, jdk17, jre8
+, buildFHSUserEnv
+# `extraJVMs` allows the user to specify additional JVMs to be made available
+# in `/opt/jvms`. This is a path MultiMC searches for Java installs, so these
+# will all be presented in the Java "auto-detect" list in MultiMC.
+# Keys in this attribute set will be used to generate the paths for each Java
+# install.
+, extraJVMs ? {}
+# It's possible for certain mods to depend on some native libraries. This
+# option allows the user to add additional libraries to the FHS environment so
+# these mods will work properly.
+, extraPkgs ? []
+}:
+let
+multimc = stdenv.mkDerivation rec {
+  name = "multimc";
+
+  src = ./.;
+
+  installPhase = ''
+    install -Dm755 $src/run.sh $out/bin/multimc
+
+    mkdir -p $out/share/pixmaps
+    install -Dm644 $src/icon.svg $out/share/pixmaps/multimc.svg
+    install -Dm755 $src/multimc.desktop $out/share/applications/multimc.desktop
+  '';
+
+  meta = with lib; {
+    description = "Free, open source launcher and instance manager for Minecraft";
```

Comment:
```suggestion
    description = "Open source launcher and instance manager for Minecraft";
 ```

Reply:
I don't appreciate this sort of poor attempt at trolling. We're here to discuss the merits of this PR, and what actual practical changes should be made to it, not to take cheap shots at people we disagree with on software licensing matters.